docs/root/start/sandboxes/index.rst

```diff
@@ -70,6 +70,7 @@ The following sandboxes are available:
     rbac
     redis
     skywalking
+    route_mirroring
```

Comment:
```suggestion
    route-mirror
```

currently there is a mix of hyphenated and underscored names in both docs and the examples, but i think lets prefer hyphenation

also - i think lets use `route-mirror` (rather than `route-mirroring`) throughout as its shorter/simpler